### ISSUE TITLE
Move quarkus-xmlsec to Quarkus CXF

### ISSUE THREAD

**Issue by ppalaga** [quarkiverse/quarkus-xmlsec#168]
Maintaining this project brings an additional overhead and Quarkus CXF is the primary user of quarkus-xmlsec anyway. 

Having said that, I propose to move the quarkus-xmlsec extension to Quarkus CXF and release/maintain it there.

Are there any concerns about it, @gastaldi @maxandersen?

**Comment by maxandersen** [comment]
Overhead in what way? 
The additional automated release or? 

How would you move it - rename artifacts/packages wouldn't be great but would then need to check how the automation works?

**Comment by gastaldi** [comment]
Based on the commits, it doesn't seem like this extension needs to be updated frequently (there is no need to update to the latest Quarkus, latest LTS is enough), so I am not sure if it's worth the trouble.

**Comment by ppalaga** [comment]
A CVE was fixed in all maintenance branches of Santuario recently. I had to upgrade in all active branches here, release them all, then upgrade in all active branches of Quarkus CXF and (tbd) Camel Quarkus upstream and midstream. Having all in Quarkus CXF would save me tons of time.

> How would you move it - rename artifacts/packages wouldn't be great 

Java packages can stay as they are, although I do not think they matter to anybody. groupId/artifactId would change.

**Comment by gastaldi** [comment]
No objections from me, if you'd like to archive this repository, just go ahead and submit a PR deleting https://github.com/quarkiverse/quarkiverse-devops/blob/main/terraform-scripts/quarkus-xmlsec.tf and removing the entry here: https://github.com/quarkiverse/quarkiverse-devops/blob/main/.github/CODEOWNERS#L121

**Comment by ppalaga** [comment]
> if you'd like to archive this repository

I do not think we can archive this repo, because we'll still have to maintain 1.1 and 2.1 branches for some time. I would just like to move the content of the main branch. I'd perhaps delete all the code and leave only the README with some explanation. WDYT?

**Comment by gastaldi** [comment]
That works too

**Comment by gastaldi** [comment]
Before proceeding I believe it would be nice to ask in the [Quarkiverse GitHub Discussions](https://github.com/orgs/quarkiverse/discussions) or in the quarkus-dev mailing list if someone objects (maybe there is a use case we haven't met yet?)

**Comment by ppalaga** [comment]
> it would be nice to ask in the [Quarkiverse GitHub Discussions](https://github.com/orgs/quarkiverse/discussions) or in the quarkus-dev mailing list

Done: https://groups.google.com/g/quarkus-dev/c/b5LsPR5V8K8
